app/code/Magento/Customer/view/frontend/templates/widget/company.phtml

```diff
@@ -0,0 +1,17 @@
+<?php
+/**
+ * Copyright © 2013-2017 Magento, Inc. All rights reserved.
+ * See COPYING.txt for license details.
+ */
+
+// @codingStandardsIgnoreFile
+
+/** @var \Magento\Customer\Block\Widget\Company $block */
+?>
+
+<div class="field company<?php if ($block->isRequired()) echo ' required' ?>">
+    <label for="company" class="label"><span><?php echo $block->escapeHtml(__('Company')) ?></span></label>
+    <div class="control">
+        <input type="text" name="company" id="company" value="<?php echo $block->escapeHtmlAttr($block->getCompany()) ?>" title="<?php echo $block->escapeHtmlAttr(__('Company')) ?>" class="input-text <?php echo $block->escapeHtmlAttr($this->helper('Magento\Customer\Helper\Address')->getAttributeValidationClass('company')) ?>">
+    </div>
+</div>
```

Comment:
Will be nice to improve formatting in templates 😉  Copy-paste code bellow.

```phtml
<div class="field company <?php echo $block->isRequired() ? 'required' : '' ?>">
    <label for="company" class="label">
        <span>
            <?php echo $block->escapeHtml(__('Company')) ?>
        </span>
    </label>
    <div class="control">
        <?php
            $_validationClass = $block->escapeHtmlAttr(
                $this->helper('Magento\Customer\Helper\Address')
                     ->getAttributeValidationClass('company')
            );
        ?>
        <input type="text"
               name="company"
               id="company"
               value="<?php echo $block->escapeHtmlAttr($block->getCompany()) ?>"
               title="<?php echo $block->escapeHtmlAttr(__('Company')) ?>"
               class="input-text <?php echo $_validationClass ?? '' ?>"
         >
    </div>
</div>
```
PS. Remember about new line at the end of file.